src/main.rs

```diff
@@ -336,13 +336,37 @@ async fn main() -> anyhow::Result<()> {
         policy_changes_topic: args.policy_changes_topic,
         policy_timeout_ms: args.policy_timeout_ms,
     };
-    let labels = args
+    let mut labels = HashMap::from([
+        ("hostcore.arch".into(), ARCH.into()),
```

Comment:
You'll probably need to add this imports here. Once again, `cargo check` is your friend!